apps/desktop/src/views/SearchView/windowSizing.ts

```
@@ -88,7 +120,8 @@ export function resolveSearchWindowHeightPolicy(
     input: SearchWindowHeightPolicyInput
 ): SearchWindowHeightPolicy {
     const hasConversationPanel = input.sessionCount > 0;
-    const hasManagedPanel = hasConversationPanel || input.quickSearchOpen;
+    const hasConversationTransition = input.conversationPending;
+    const hasManagedPanel = hasConversationPanel || input.quickSearchOpen || hasConversationTransition;
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)